core/test/forest/ForestCharacterizationTest.cpp

```diff
@@ -61,7 +61,7 @@ void update_predictions_file(const std::string file_name, std::vector<Prediction
   FileTestUtilities::write_csv_file(file_name, values);
 }
 
-TEST_CASE("quantile forest predictions have not changed", "[quantile], [characterization]") {
+TEST_CASE("quantile forest predictions have not changed", "[quantile, characterization]") {
```

Comment:
Actually... are you sure this is right? Now our automated travis tests are failing, because the characterization tests aren't being properly excluded when running this line: `./build/grf exclude:[characterization]`